package.json

```diff
@@ -35,6 +35,11 @@
       "prettier --parser typescript --write"
     ]
   },
+  "prettier": {
```

Comment:
Please remove the prettier config here, let's just roll with the default config.